tools/spectral/Makefile

```diff
@@ -0,0 +1,32 @@
+# A Self-Documenting Makefile: http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
```

Comment:
do we still need this file if we are running spectral via the GH action?

Reply:
I'd like to keep for local development